src/cli/run.rs

```diff
@@ -202,8 +202,22 @@ pub async fn execute_script_with_output(
 pub async fn execute(args: Args) -> miette::Result<()> {
     let project = Project::load_or_else_discover(args.manifest_path.as_deref())?;
 
+    // Split 'task' into arguments if it's a single string, supporting commands like:
+    // "test 1 == 0 || echo failed" or "echo foo && echo bar".
+    // This prevents shell interpretation of pixi run inputs.
+    // Use as-is if 'task' already contains multiple elements.
+    let task = if args.task.len() == 1 {
+        args.task[0]
+            .split_whitespace()
```

Comment:
What happens if you pass 'echo "HELLO WORLD!"'? 🤔

Reply:
Maybe you should shlex::split? https://docs.rs/shlex/latest/shlex/fn.split.html

Reply:
This is already parsed as two things and the `pixi run "echo 'Hello World'"` also works but that becomes weird. 

But to miss edge cases we didn't think of now that shlex lib looks like a beter way forward!